release.yaml

```diff
@@ -23,8 +23,10 @@ neuvector-monitor:
   - 105.0.0+up2.8.3
 rancher-cis-benchmark:
   - 105.0.0+up7.0.0
+  - 105.0.1+up7.0.1
 rancher-cis-benchmark-crd:
   - 105.0.0+up7.0.0
```

Comment:
@nicholasSUSE this version is already released right? shouldnt it be removed already?

Reply:
Yes it should be removed but it does not break anything. 
I will be removing it.

Reply:
Done and thanks for reviewing!